src/main/java/org/terracotta/statistics/ConstantValueStatistic.java

```diff
@@ -24,17 +24,18 @@
  */
 public class ConstantValueStatistic<T extends Number> implements ValueStatistic<T> {
 
-  private static final Map<Object, ValueStatistic<?>> common = new HashMap<Object, ValueStatistic<?>>();
+  private static final Map<Object, ValueStatistic<Number>> common = new HashMap<>();
```

Comment:
To me the wildcard here is correct.  The semantics are that the map can contain ValueStatistics of any type not just ValueStatistics of type Number.  I do understand that the current semantics are the same because Number is the bound on ValueStatistic - a good counter here is that `?` is still correct if the bound on ValueStatistic changes... but using `Number` would require this to change with the ValueStatistic declaration.

Reply:
I don't get it. `instance` will always return a `ValueStatistic<T>`. And `T extends Number`. So it will be a `Number`. But then, I don't care. I will still need to suppress the warning.

Reply:
I've turned it back to ?